src/sass/components/_header.scss

```diff
@@ -188,13 +188,23 @@ header {
 
         select {
           -webkit-appearance: none;
+          appearance: none;
           border: 0;
           background-color: transparent;
-          outline: none;
+          outline: 0;
           padding: 5px 30px 5px 35px;
           font-size: 14px;
           position: relative;
           z-index: 1;
+          white-space: nowrap;
+        }
+
+        &:hover {
+          background-color: $primary;
+          i:first-child,  i:last-child {
```

Comment:
proszę zamień to na 
i:first-child,
i:last-child {

łatwiej rozwiązywać wtedy konflikty, gdyby się zdarzyły w tej linii :)